src/Starlight/Actor.hs

```diff
@@ -27,31 +30,34 @@ import Unit.Mass
 import Unit.Time
 
 data Actor = Actor
-  { position  :: !(V3 (Kilo Metres Float))
-  , velocity  :: !(V3 ((Kilo Metres :/: Seconds) Float))
-  , rotation  :: !(Quaternion Float)
-  , mass      :: !(Kilo Grams Float)
-  , magnitude :: !(Kilo Metres Float) -- approx. equivalent to diameter; should bound the actor’s geometry
+  { position  :: !(V3 (Mega Metres Double))
+  , velocity  :: !(V3 ((Mega Metres :/: Seconds) Double))
+  , rotation  :: !(Quaternion Double)
+  , mass      :: !(Kilo Grams Double)
+  , magnitude :: !(Mega Metres Double) -- approx. equivalent to diameter; should bound the actor’s geometry
```

Comment:
Switching from km to Mm and `Float` to `Double` was enough to make almost everything buttery-smooth out at least as far Jupiter.